source/03-components/mega-menu/mega-menu.scss

```diff
@@ -2,6 +2,16 @@
 // Styles for the Mega Menu.
 @use '00-config' as *;
 
+// Adjust the selector and colors as needed for your site.
+// If the mega menu is not the top element on the page,
+// the element(s) above it need a background color and
+// z-index in order to cover the mega menu.
+.l-header {
+  background: gesso-color('background', 'site');
+  position: relative;
+  z-index: 2;
+}
+
```

Comment:
I like the documentation here. We should move the `l-header` styles to the header layout partial.

Reply:
@dcmouyard I thought about that but intentionally did **not** put the styles there because I only want them to apply when the mega-menu CSS file/mega-menu library is included. I'm assuming a developer will move them around as needed when adjusting the styles for their individual site. I also considered not putting any default styles on the header, but the mega menu looks broken on first enabling otherwise and it's likely not going to be clear to a developer how to fix it. So this seemed like the best compromise to me. Prefer to leave as-is.

Reply:
@kmonahan Ah, I had forgotten that this is not a partial that will go into `styles.css` so including it here makes more sense. 👍